packages/uiweb/src/lib/hooks/chat/usePushChatStream.ts

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+import { useEffect, useState } from 'react';
+import { CONSTANTS, PushAPI, SignerType } from '@pushprotocol/restapi';
+import { ENV } from '../../config';
+import { useChatData } from './useChatData';
+
+
+export type PushChatStream = {
+    account?: string | null;
+    env?: ENV;
+};
+
+export const pushChatStream = () => {
+
+
+    const {
+        account,
+        isPushChatStreamConnected,
+        setIsPushChatStreamConnected,
+        env,
+        pushUser
+
+    } = useChatData();
+
+
+    const [chatStream, setChatStream] = useState<any>({}) // to track any new messages
+    const [chatRequestStream, setChatRequestStream] = useState<any>({}); // any message in request
+    const [groupMetaStream, setGroupMetaStream] = useState<any>({}); //group info
+    const [pushChatStream, setPushChatStream] = useState<any>({}); //stream connection
```

Comment:
Any specific reason we shifted the stream locally instead of keeping it in context?